src/components/common/layout/base.njk

```diff
@@ -59,12 +59,14 @@
 
 {% block main %}
     <div class="govuk-width-container {{ containerClasses }}">
+        {% if webChannel %}
```

Comment:
See [comment on related PR](https://github.com/govuk-one-login/authentication-frontend/pull/2114/files#r1786608393) about inverting the logic